realm/chess.gno

```diff
@@ -229,6 +227,11 @@ func newGame(caller, opponent std.Address, seconds, increment int) *Game {
 	var games []*Game
 	if exist {
 		games, _ = val.([]*Game)
+
+		// Close previous games that timed-out
+		for _, g := range games {
+			_, _ = claimTimeout(g.ID)
```

Comment:
better if we can have the unexported `claimTimeout` accept directly a `*Game` (this way we don't need to re-fetch Game again from gameStore, seeing as they are the same pointer it is not necessary